decidim-admin/app/models/decidim/admin/abilities/base.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    module Abilities
+      # Defines the abilities for a user in the admin section. Intended to be
+      # used with `cancancan`.
+      class Base
```

Comment:
We already had an `Admin` ability, I renamed it to `AdminUser` and made it extend the base `Decidim::Ability` class. Do you want me to undo the renaming? What should this `Admin` ability class that you say we should have?